test/test_filter_process_ucs_syslog.rb

```diff
@@ -37,7 +37,7 @@ def getPassword()
             def callUcsApi(host, body)
                 if body.delete(' ') == "<aaaLogin inName=\"testDomain\\testUsername\" inPassword=\"testPassword\"></aaaLogin>".delete(' ') && host == "1.1.1.1"
                     return '<aaaLogin cookie="" response="yes" outCookie="1111111111/12345678-abcd-abcd-abcd-123456789000"> </aaaLogin>'
-                elsif body.delete(' ') == "<configResolveDn cookie=\"1111111111/12345678-abcd-abcd-abcd-123456789000\" dn=\"sys/chassis-4/blade-7\"></configResolveDn>".delete(' ') && host == "1.1.1.1"
+                elsif body.delete(' ') == "<configResolveDn cookie=\"1111111111/12345678-abcd-abcd-abcd-123456789000\" dn=\"sys/chassis-4/blade-7\"></configResolveDn>".delete(' ') && (host == "1.1.1.1" || host == "1.1.1.2")
                     return '<lsServer assignedToDn="org-root/org-T100/ls-testServiceProfile"/>'
                 elsif body.delete(' ') == "<configResolveDn cookie=\"1111111111/12345678-abcd-abcd-abcd-123456789000\" dn=\"sys/chassis-4/blade-5\"></configResolveDn>".delete(' ') && host == "1.1.1.1"
```

Comment:
Why didn't this line's condition on the IP change as well?

Reply:
so the idea is that in the tests, if you pass in this specific body along with host of 1.1.1.1 OR 1.1.1.2, it will return a valid response. However, later in the update etcd section (which is after the callUcsApi call) if you pass in host that is NOT 1.1.1.1 it will give you an error.

This change is to let host 1.1.1.2 pass in the callUcsApi call, but fail in the updateEtcd call.

Reply:
ack